pipelines/README.md

```diff
@@ -93,6 +93,6 @@ oc create -f test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow
 > If using one of the provided `PipelineRun` files, make sure all of the parameters match your Git environment.
 
 ```bash
-oc apply -k gitops-update-pipeline/
-oc create -f gitops-update-pipeline/gitops-update-pipelinerun-tensorflow-housing.yaml
+oc apply -k tekton/gitops-update-pipeline/
+oc create -f tekton/gitops-update-pipeline/gitops-update-pipelinerun-tensorflow-housing.yaml
```

Comment:
You added `tekton/` here but left the others as is? This also contradicts the notice slightly higher in the README that says to `cd` into `tekton` first